### ISSUE TITLE
YoloV8-Seg

### ISSUE THREAD

**Issue by rhalder2023** [ultralytics/ultralytics#17343]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

On your github landing page, i noticed the pretrained coco model weights are only available for yolo-11, where do i get the yolov8-seg model pretrained on coco weights? I cant seem to find them

**Comment by UltralyticsAssistant** [comment]
👋 Hello @rhalder2023, thank you for reaching out and your interest in Ultralytics 🚀! For new users, we recommend checking out the [Docs](https://docs.ultralytics.com) where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) examples which might have the answers you're looking for.

Regarding your query about YOLOv8-Seg and its pretrained COCO model weights, if this is a 🐛 Bug Report or you're unable to locate resources, we kindly ask you to provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) so we can assist you better.

For further information on custom training processes, consider checking out our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/). Make sure to share any dataset image examples or training logs you're working with to enable us to provide more tailored support.

Join the Ultralytics community to exchange ideas and get assistance quickly. You can join us on [Discord](https://ultralytics.com/discord) for real-time chat 🎧, engage in detailed discussions on [Discourse](https://community.ultralytics.com), or mingle with fellow users on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

To make sure you have the latest bug fixes and features, please upgrade to the newest `ultralytics` package and verify you meet all the [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) within a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO models can be run in various verified environments with the following options for free GPU usage and pre-installed essential dependencies:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

This badge indicates the current status of Ultralytics CI tests. A green badge confirms that all tests are passing and verifying correct operation of YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu environments every 24 hours and on every commit.

This is an automated response, but rest assured, an Ultralytics engineer will review your question and offer further help soon.

**Comment by rhalder2023** [comment]
> ### Search before asking
> * [x]  I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.
> 
> ### Question
> On your github landing page, i noticed the pretrained coco model weights are only availab
Okay can i get it from the models page in your docs?

**Comment by Y-T-G** [comment]
You can find it in the models page.

https://docs.ultralytics.com/models/yolov8/

**Comment by rhalder2023** [comment]
Thanks

**Comment by glenn-jocher** [comment]
You're welcome! If you have any more questions or need further assistance, feel free to ask.
